blog.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <div class="page-wrapper">
+      <header class="heder-container">
+        <a class="logo" href="./index.html">
+          <img src="./images/logo.svg" alt="LOGO"
+        /></a>
+        <nav class="header-nav">
+          <ul class="header-nav-container">
+            <li class="header-nav-item">
+              <a href="./index.html">John's home page</a>
+            </li>
+            <li class="header-nav-item"><a href="./blog.html">Blog</a></li>
+            <li class="header-nav-item"><a href="./work.html">Work</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+        <div class="blog-title-container"><h1 class="blog-title">Blog</h1></div>
```

Comment:
You should separate each logical or content or business block on the page by HTML tag `section`

Reply:
?